examples/react-use-awareness/src/App.jsx

```diff
@@ -1,30 +1,21 @@
-import {
-  useDocument,
-  useLocalAwareness,
-  useRemoteAwareness,
-} from "@automerge/react"
+import { useDocument, usePresence } from "@automerge/react"
```

Comment:
Note: I combined the two hooks into one, since they need to communicate (local awareness needs to broadcast when a new peer is seen). The current code does that with a shared global, but that has a bunch of shortcomings: e.g., it could lead to unnecessary messages if working with presence on two unrelated handles (or even repos). We could use React Context instead, but I don't think that's better.